deploy/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: "3.9"
+
+services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - "127.0.0.1:${INTERNAL_HOST_PORT:-1235}:80"
+    restart: always
```

Comment:
```suggestion
    restart: unless-stopped
```

Reply:
Use it for all `restart:` sections. We do not want to start editor on restart of computer if we stopped it.

Reply:
BTW, i do not see `restart:` section anywhere  ... it should be part of each service, please add it there

Reply:
done